### PR TITLE
Add tolerations support to Helm Chart

### PR DESCRIPTION
**Pull request by denis111**
# Description

Added tolerations support to values of Helm Chart. There was only nodeSelector available but we can't schedule pods on tainted nodes with with only nodeSelector. Most of other helm charts around have tolerations support through values.

## Issue reference

Please reference the issue this PR will close: #3534

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

I've already applied tolerations in this PR on our production Dapr.

### CODE REVIEW

**Comment by daixiang0** [comment]
lgtm

**Comment by denis111** [comment]
"E2E tests on KinD / e2e" fail is not my fault ;)

**Review by daixiang0 — DISMISSED** [review]

**Comment by denis111** [comment]
I see that the chart in master branch is behind the one in release-1.4, how we can fix it?

**Comment by daixiang0** [comment]
Not sure the best way to do it, /cc @artursouza

**Comment by yaron2** [comment]
@denis111 please resolve conflicts.

File: charts/dapr/Chart.yaml [hunk 1]
```diff
@@ -2,23 +2,23 @@ apiVersion: v1
 appVersion: "1.4.0"
 description: A Helm chart for Dapr on Kubernetes
 name: dapr
-version: 1.4.0
```
Comment (artursouza) [inline charts/dapr/Chart.yaml:5]: These are only updated during release, please, revert.

File: charts/dapr/charts/dapr_dashboard/Chart.yaml [hunk 2]
```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: "1.0"
 description: A Helm chart for Dapr Dashboard
 name: dapr_dashboard
-version: 0.8.0
+version: 0.8.1
```
Comment (artursouza) [inline charts/dapr/charts/dapr_dashboard/Chart.yaml:5]: These are only updated during release, please, revert.

File: charts/dapr/charts/dapr_operator/Chart.yaml [hunk 3]
```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: "1.0"
 description: A Helm chart for Dapr Kubernetes Operator
 name: dapr_operator
-version: 1.4.0
+version: 1.4.1
```
Comment (artursouza) [inline charts/dapr/charts/dapr_operator/Chart.yaml:5]: These are only updated during release, please, revert.

File: charts/dapr/charts/dapr_placement/Chart.yaml [hunk 4]
```diff
@@ -2,5 +2,5 @@ apiVersion: v1
 appVersion: "1.0"
 description: A Helm chart for Dapr Kubernetes placement
 name: dapr_placement
-version: 1.4.0
+version: 1.4.1
```
Comment (artursouza) [inline charts/dapr/charts/dapr_placement/Chart.yaml:5]: These are only updated during release, please, revert.

File: charts/dapr/charts/dapr_sentry/Chart.yaml [hunk 5]
```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: "1.0"
 description: A Helm chart for Dapr Sentry
 name: dapr_sentry
-version: 1.4.0
+version: 1.4.1
```
Comment (artursouza) [inline charts/dapr/charts/dapr_sentry/Chart.yaml:5]: These are only updated during release, please, revert.

File: charts/dapr/charts/dapr_sidecar_injector/Chart.yaml [hunk 6]
```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: "1.0"
 description: A Helm chart for the Dapr sidecar injector
 name: dapr_sidecar_injector
-version: 1.4.0
+version: 1.4.1
```
Comment (artursouza) [inline charts/dapr/charts/dapr_sidecar_injector/Chart.yaml:5]: These are only updated during release, please, revert.

**Review by artursouza — CHANGES_REQUESTED** [review]

**Comment by denis111** [comment]
@artursouza Ok, done. But I've just followed helm best practices for semver...

**Review by cvictory — DISMISSED** [review]
LGTM

**Comment by denis111** [comment]
@artursouza should I do anything more or this PR is ready to be approved?

**Review by artursouza — APPROVED** [review]

**Pull request merged**